src/core/diags/factory_diags.cpp

```diff
@@ -271,6 +272,7 @@ Error Diags::ProcessSend(uint8_t aArgsLength, char *aArgs[], char *aOutput, size
 
     SuccessOrExit(error = ParseLong(aArgs[1], value));
     VerifyOrExit(value <= OT_RADIO_FRAME_MAX_SIZE, error = kErrorInvalidArgs);
+    VerifyOrExit(value >= (OT_RADIO_FRAME_MIN_SIZE), error = kErrorInvalidArgs);
```

Comment:
nit: to be consistent with line above
```suggestion
    VerifyOrExit(value >= OT_RADIO_FRAME_MIN_SIZE, error = kErrorInvalidArgs);
```